pages/index.vue

```diff
@@ -138,6 +139,24 @@ export default defineComponent({
        * @hints use store and set loading true
        * @caution you have to assign new value to todos for it to update
        */
+      const headers = {
+        Authorization: 'Token ' + this.$store.getters.token,
+      }
+      this.$axios
+        .get('todo/', { headers })
+        .then(({ data }) => {
+          this.todos = []
+          if (data) {
+            for (let element of data) {
+              element.editing = false
+              this.todos.push(element)
```

Comment:
Although this works for vuejs frameworks like svelte wont watch changes to array using push so its sometimes recommended to just assign and new array using `map` but this works

Reply:
Thank you, I'll take a look at that also and try to modify the code.